ContextReinstatement.ts

```
@@ -0,0 +1,1773 @@
+    return memory.cognitiveState?.cognitiveLoad * 10; // MB approximation
+  }
+
+  private determineOperationalMode(memory: TemporalMemoryItem): string {
+    if (memory.memoryType === 'procedural') return 'execution-mode';
+    if (memory.cognitiveState?.focusLevel > 8) return 'focused-analysis';
+    if (memory.cognitiveState?.emotionalIntensity > 7) return 'emotional-processing';
+    return 'general-operation';
+  }
+
+  private extractActiveOperations(memory: TemporalMemoryItem): string[] {
+    return [memory.topic, ...memory.cognitiveState?.mentalContext.solutionPath];
```

Biome (2.1.2)

[error] Unsafe usage of optional chaining.

If it short-circuits with 'undefined' the evaluation will throw TypeError here:


(lint/correctness/noUnsafeOptionalChaining)